1-exercises/A-array-find/exercise.js

```diff
@@ -4,6 +4,9 @@
 */
 
 // write your code here
+function findLongNameThatStartsWithA(name) {
+  return name[0]==='A' && name.length>7;
```

Comment:
here you need to use find method to get the the name but your code returns false instead